src/util/sync.ts

```diff
@@ -225,7 +223,7 @@ export const sync = (thoughtIndexUpdates = {}, contextIndexUpdates = {}, { local
     logWithTime('sync: localPromises complete')
 
     // firebase
-    if (isDocumentEditable() && remote) {
+    if (process.env.NODE_ENV !== 'test' && isDocumentEditable() && remote) {
```

Comment:
If `remote` is true, `process.env.NODE_ENV !== 'test'` will always be true, so you can remove it.